### PR TITLE
fix text split when creating tags

### PR DESCRIPTION
**Pull request by steveoni**
this pr fix text split when creating single tags

Screenshot 2022-03-03 at 14 43 27

Screenshot 2022-03-03 at 14 43 49

Screenshot 2022-03-03 at 14 44 51

### CODE REVIEW

File: ckanext/vocabulary/blueprints/vocabulary.py [hunk 1]
```diff
@@ -119,6 +124,14 @@ def new_tags(id):
...
                 tag_dict['vocabulary_id'] = id
                 get_action('tag_create')(context, tag_dict)
+        else:
+            tag_dict = {
+                'name': en_tags,
+                'name_translated-en': en_tags,
+                'name_translated-ar': ar_tags,
+                'vocabulary_id': id
+            }
+            get_action('tag_create')(context, tag_dict)
```
Comment (sagargg) [inline ckanext/vocabulary/blueprints/vocabulary.py]: @steveoni  can we display success and error flash messages.  something like 

``` python
try:
      get_action('tag_create')(context, tag_dict)
      h.flash_success(u"Tag sucessfully added to the category.")
except ValidationError as e:
      h.flash_error(e.error_dict)
```

File: ckanext/vocabulary/blueprints/vocabulary.py [hunk 2]
```diff
@@ -109,7 +114,7 @@ def new_tags(id):
         ar_tags = params['ar']
         en_tags = params['en']
 
-        if ar_tags and en_tags:
+        if (ar_tags and en_tags) and (type(ar_tags) is list):
             ztags = zip(en_tags, ar_tags)
             tags = []
```
Comment (sagargg) [inline ckanext/vocabulary/blueprints/vocabulary.py]: looks like tags = [] is  not in used

File: ckanext/vocabulary/blueprints/vocabulary.py [hunk 3]
```diff
@@ -58,7 +58,12 @@ def new():
...
                 tags.append(tag_dict)
-
+        else:
+            tags = [{
+                'name': en_tags,
+                'name_translated-en': en_tags,
+                'name_translated-ar': ar_tags
+            }]
         data_dict['tags'] = tags
         create_vocab = get_action('vocabulary_create')(context, data_dict)
```
Comment (sagargg) [inline ckanext/vocabulary/blueprints/vocabulary.py]: Can we display flash messages on success or when errors happen.

**Review by sagargg — CHANGES_REQUESTED** [review]

**Comment by sagargg** [comment]
@steveoni  can reduce the button margin for the label. 

Screen Shot 2022-03-03 at 8 56 44 PM

**Comment by sagargg** [comment]
Same on this page also, reduce button margin. It looks  little bit untidy. 
Screen Shot 2022-03-03 at 9 00 48 PM

**Comment by osahon-okungbowa** [comment]
@steveoni i agree with @sagargg . Also, please reduce the font size of the "Add Tag" headings. It should be slightly smaller than the main heading

**Comment by steveoni** [comment]
@osahon-okungbowa and @sagargg thanks for the comment and pointing that out. here is an update.

Screenshot 2022-03-04 at 09 59 20

Screenshot 2022-03-04 at 10 10 27



Screenshot 2022-03-04 at 10 00 01

**Comment by sagargg** [comment]
@steveoni  Tags filed alignment is little compared to other fields.   
 
Screen Shot 2022-03-04 at 2 55 22 PM

**Comment by steveoni** [comment]
> @steveoni Tags filed alignment is little compared to other fields.
> 
> Screen Shot 2022-03-04 at 2 55 22 PM

is a different section of it self and it think is okay

**Comment by sagargg** [comment]
@steveoni  okay. I'll merge this for now.

**Comment by sagargg** [comment]
**LGTM**

**Pull request merged**